lms/templates/course_modes/unfbe.html

```diff
@@ -0,0 +1,38 @@
+## This template is for track selection with partial or no Feature Based
+## Enrollment (FBE). In other words, it is un-FBE. FBE is when the learner in
+## the default audit track has a limited time to complete the course (course
+## access duration) and has some of the content of the course unavailable (gated
+## content). This template covers un-FBE, which is when the learner is subject
+## to either: (1) gated content but not course access duration; (2) course
+## access duration but not gated content; or (3) neither gated content nor
+## course access duration.
+
+<%page expression_filter="h"/>
+<%inherit file="track_selection.html" />
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
+%>
+<%block name="track_selection_certificate_bullets">
+<div class="choice-bullets track-selection-type-unfbe">
+    <ul>
+        <li>${Text(_("Showcase a {link_start}verified certificate{link_end} of completion on your resumé to advance your career")).format(
+            link_start=HTML('<b><u><a class="verified" href="https://www.edx.org/verified-certificate" target="_blank">'),
```

Comment:
Can you add the non-hardcoded version here too?

Reply:
Yep, thanks for the catch!